### ISSUE TITLE
Portfolio Review

### ISSUE THREAD

**Issue by baileym13** [Albertsofc/csPortfolio#1]
Your portfolio is very organized and includes a lot of features! I enjoy how you had a history of your time in Computer Science, and all your responses were very well written. I liked the italics you used and the boxes and Bolded letters, you clearly know your way around markdown. There are a couple spelling errors, so you should look over what you have written, and one of your links is not live. Could you add links to things you've done for Hack Club perhaps, maybe some pictures to make it more lively? Bravo!

Madison Bailey

**Comment by sydneyessler** [comment]
I agree. I think adding more information about hack club would be great too!
